### DISCUSSION TITLE
[docs][web-components]: add property assignment script to code snippet

### DISCUSSION THREAD

**Discussion by bennypowers** [storybookjs/storybook#23115] [Ideas]
We should think about how to handle properties as for some cases the source wouldn't really show the state of the component.  This is a little different from frameworks as the difference between attributes and properties are abstracted away.  My initial idea is to a script tag that populates properties.

_Originally posted by @brion-fuller in https://github.com/storybookjs/storybook/pull/15337#discussion_r667022651_

**Comment by break-stuff** [comment]
This is something that I have set up in my [storybook helpers](https://www.npmjs.com/package/wc-storybook-helpers), but would like to get added to the storybook repo eventually. It splits the properties into their own category and has data binding set up for the controls (if you are using the `template` function).
